### DISCUSSION TITLE
坐标轴label过长时，是否可以换行

### DISCUSSION THREAD

**Discussion by morelearn1990** [antvis/G2#5146] [Q&A] [answered]
##### Checklist

- [x] `npm test` passes
- [x] benchmarks are included
- [x] commit message follows commit guidelines
- [x] documents are updated

##### Description of change

坐标轴label过长时，是否可以换行

**Comment by hustcc** [comment, marked as answer]
加上 `\n` 就好。

**Comment by morelearn1990** [comment]
谢谢，发现了。正准备关掉 issue ，变成 discussions 怎么关掉呀？
